slick/src/main/scala/slick/jdbc/MySQLProfile.scala

```diff
@@ -94,6 +94,12 @@ trait MySQLProfile extends JdbcProfile { profile =>
         else l
       }
     }
+
+    //Reference: https://github.com/slick/slick/issues/1419
+    override def createTableNamer(meta: MTable): TableNamer = new TableNamer(meta){
+      override def schema = meta.name.catalog
+      override def catalog = meta.name.schema 
```

Comment:
Yeah, I'd like to better understand what's at fault here first. Is it the the mysql's jdbc driver returning meta data differently than others?

Also as always, would be great to have a test for this :)